homework/index.js

```diff
@@ -23,28 +23,73 @@
       if (key === 'text') {
         elem.textContent = value;
       } else {
-        elem.setAttribute(key, value);
+        elem.setAttribute(key, value); // ?
       }
     });
     return elem;
   }
 
+  function changeDateTimeFormat(dateTime) {
+    const timeFormat = new Date(dateTime);
+    return timeFormat.toLocaleString();
+  }
+
   function renderRepoDetails(repo, ul) {
-    createAndAppend('li', ul, { text: repo.name });
+    const list = createAndAppend('li', ul, { class: 'topClass' });
+    const name = createAndAppend('li', list, {
```

Comment:
You can not have an `li` inside another `li`. This is invalid HTML (although the browser will try and make the best of it). An `li` can only be a child of an `ol` or `ul` (or the less used `menu`). See:

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/li

This is the HTML you have generated:

![image](https://user-images.githubusercontent.com/2788771/65320331-eafa4a80-dba1-11e9-827b-97b9eec6d747.png)